frontend/src/patientApp/timeOfTest/TestResult.tsx

```diff
@@ -0,0 +1,110 @@
+import { connect, useSelector } from "react-redux";
+
+import { formatFullName } from "../../app/utils/user";
+import { capitalizeText } from "../../app/utils/text";
+
+const TestResult = () => {
+  const patient = useSelector((state) => (state as any).patient as any);
```

Comment:
`useSelector` takes a couple generics: the state type and the expected return type. We now have `RootState` exported from `store.ts`. I am less confident what the return type here is, hopefully `Patient`

```typescript
const patient = useSelector<RootState, Patient>((state) => state.patient);
```